octobot_trading/exchanges/connectors/ccxt/ccxt_websocket_connector.py

```diff
@@ -909,7 +916,8 @@ async def orders(self, orders: list, **kwargs):
         """
         # TODO update this when supported (ccxt is supporting it)
         raise NotImplementedError("orders callback is not implemented")
-        adapted = [self.adapter.adapt_order(order) for order in orders]
+        symbol = None #todo
```

Comment:
should we keep it?

Reply:
as long as it's raising NotImplementedError yes